src/node/resolver.ts

```diff
@@ -24,10 +24,15 @@ export interface InternalResolver {
 
 const defaultRequestToFile = (publicPath: string, root: string): string => {
   if (moduleRE.test(publicPath)) {
-    const moduleFilePath = idToFileMap.get(publicPath.replace(moduleRE, ''))
-    if (moduleFilePath) {
-      return moduleFilePath
+    const id = publicPath.replace(moduleRE, '')
+    const cachedModuleFilePath = idToFileMap.get(id)
+    if (cachedModuleFilePath) {
+      return cachedModuleFilePath
     }
+    const resolved = resolveNodeModuleFile(root, id)
+    if (!resolved) throw new Error(`Can't resolve "${id}"`)
```

Comment:
I doesn't make sense to continue, so I throw here. Is there a better choice?

Reply:
It will eventually error elsewhere, it's better to let it pass through.